modules/client/main.tf

```diff
@@ -89,7 +89,6 @@ resource "google_compute_instance" "forseti-client" {
   dynamic "network_interface" {
     for_each = local.network_interface
     content {
-      address            = lookup(network_interface.value, "address", null)
```

Comment:
Can you add a comment about why this was removed? What is the impact of removing this? Will this cause any breaking changes? Same with the RTE and server modules.

Reply:
Done. There is no impact. `network_ip` is supported to replace `address`, and address was probably kept for backwards compatibility and not removed. `address` was deprecated since google provider 2.0.0, and got removed starting 3.0.0.

And actually, currently it seems neither `address` nor `network_ip` is used at all by the module. So there shouldn't be any impact.